src/pyvesync/vesync.py

```diff
@@ -56,6 +58,7 @@ class VeSync:  # pylint: disable=function-redefined
         '_token',
         '_verbose',
         'country_code',
+        'region',
         'enabled',
         'in_process',
         "language",
```

Comment:
What do you think about adding an optional argument for region? So that it can be specified instead of trying and failing first.

Reply:
Actually, I just found an approach that doesn't need a separate country code and region, so it would work similar to the VeSync app by specifying the country.